builder/views.py

```diff
@@ -197,6 +198,11 @@ def update(request, draft):
 @require_permission
 def new_search(request, draft):
     term = request.POST["search"].strip()
+    # Ensure that the term is not an empty string after slugifying
+    # (e.g. if the user entered "*" as a search term)
+    if not slugify(term):
+        messages.info(request, f'"{term}" is not a valid search term')
+        return redirect(draft)
```

Comment:
This will redirect to `codelists.views.version`, which will redirect to `builder.views.draft` -- you could skip a redirect by redirecting to `draft.get_builder_draft_url()`.